dotCMS/src/main/java/com/dotcms/rendering/velocity/directive/TemplatePathStrategyResolver.java

```diff
@@ -137,31 +138,39 @@ private boolean isPath (final String templatePath) {
         }
 
         private String getPath(final RenderParams params, final String path, final String uid) {
-
-
-            final Host host     = params.currentHost;
-            String templatePath = path;
-
-
             try {
-
-                templatePath =(!templatePath.startsWith(HOST_INDICATOR))?
-                        builder(FORWARD_SLASH, params.mode.name(), FORWARD_SLASH, HOST_INDICATOR,
-                            this.getHost(host).getHostname(), path.startsWith(FORWARD_SLASH)? StringPool.BLANK:FORWARD_SLASH,
-                            path, FORWARD_SLASH, uid, PERIOD, VelocityType.CONTAINER.fileExtension).toString():
-
-                        builder(FORWARD_SLASH, params.mode.name(), FORWARD_SLASH,
-                            path, FORWARD_SLASH, uid, PERIOD, VelocityType.CONTAINER.fileExtension).toString();
+                return FileAssetContainerUtil.getInstance().isFullPath(path) ?
+                        getContainerResourcePathFromFullPath(params, path, uid) :
+                        getContainerResourceFromRelativePath(params, path, uid);
             } catch (Exception e) {
 
-                Logger.warn(this.getClass(), " - unable to resolve " + templatePath + " getting this: "+ e.getMessage() );
+                Logger.warn(this.getClass(), " - unable to resolve " + path + " getting this: "+ e.getMessage() );
                 if(e.getStackTrace().length>0) {
                     Logger.warn(this.getClass(), " - at " + e.getStackTrace()[0]);
                 }
                 throw new DotStateException(e);
             }
+        }
+
+        @NotNull
```

Comment:
remove this, won't work on java 11

Reply:
done https://github.com/dotCMS/core/pull/18033/commits/286b9c09f52d6e0bb78ec9761ec3000c06d7f38a#diff-852b80877a8a40dd946126a363d2dc76L155